packages/e2e-tests/specs/experiments/blocks/avatar.test.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	clickBlockAppender,
+	createNewPost,
+	getEditedPostContent,
+	insertBlock,
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Avatar block', () => {
+	beforeEach( async () => {
+		await createNewPost();
+	} );
+
+	it( 'can be inserted', async () => {
+		// Create a block with some text that will trigger a list creation.
+		await insertBlock( 'Avatar' );
+		expect( await getEditedPostContent() ).toMatchSnapshot();
+	} );
+
+	it( 'can be created by typing "/avatar"', async () => {
+		// Create a list with the slash block shortcut.
+		await clickBlockAppender();
+		await page.keyboard.type( '/avatar' );
+		await page.waitForXPath(
+			`//*[contains(@class, "components-autocomplete__result") and contains(@class, "is-selected") and contains(text(), 'Avatar')]`
+		);
+		await page.keyboard.press( 'Enter' );
+		expect( await getEditedPostContent() ).toMatchSnapshot();
+	} );
+} );
```

Comment:
By a similar reasoning as over at https://github.com/WordPress/gutenberg/pull/39502#discussion_r830318752, I'd remove the snapshot tests TBH -- which currently means removing the e2e tests altogether 😅  (We also have some basic coverage by the fixtures in `test/integration/fixtures/blocks/core__avatar.*`.)

I think an e2e test for this block could maybe check if setting the avatar to a different user really shows the correct image for that user. Let's maybe file an issue for an e2e test to keep track of it, and implement it later.

Reply:
Filed issue to request e2e coverage: https://github.com/WordPress/gutenberg/issues/39645